inform.py

```diff
@@ -195,7 +198,20 @@ def handle(self, signum, _stack_frame):
 
 
 def main():
-    conf = toml.load('conf.toml')
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-c", "--conf", help="Configuration file; (default: ./conf.toml)", default="conf.toml")
+    parser.add_argument("-p", "--port", help="Local port to listen to; (default: 8080)", default=8080, type=int)
+    args = parser.parse_args()
+
+    conf_file = args.conf
+    port = args.port
+
+    try:
+        conf = toml.load(conf_file)
+    except Exception as _string:
```

Comment:
actually no need to catch the exception, just leave it to python to print the exception, stack trace & exit with return code != 0.

(and btw, exception is not a string in python)